chainer/optimizer.py

```diff
@@ -652,9 +655,58 @@ def serialize(self, serializer):
             if rule is not None:
                 rule.serialize(serializer[name])
 
+    def loss_scaling(self, interval=1000, scale=None):
+        """Sets loss scaling configurations.
```

Comment:
A minor but how about
```suggestion
        """Configures the loss scaling algorithm.
```